app/src/Tasks/index.js

```diff
@@ -22,7 +22,7 @@ const Posts = () => {
   };
 
   const fetchColorData = () => {
-    let url = "http://colormind.io/api/";
+    let url = "https://colormind.io/api/";
```

Comment:
Suggestion: Instead of defining a variable as URL, you can also pass the URL directly in your fetch function. Saves you a line of code!